tests/MySqlConnector.Tests/MySqlConnectionStringBuilderTests.cs

```diff
@@ -26,6 +26,7 @@ public void Defaults()
 #endif
 			Assert.Equal(15u, csb.ConnectionTimeout);
 			Assert.False(csb.ConvertZeroDateTime);
+			Assert.Equal(DateTimeKind.Unspecified, csb.DateTimeKind);
```

Comment:
Needs to be in a `#if !BASELINE` block.

Reply:
Oops, forgot to run the BASELINE tests, thanks.